packages/components/src/components/input/input.tsx

```diff
@@ -22,7 +22,7 @@ export type TInputProps = {
     input_id?: string;
     is_relative_hint?: boolean;
     label_className?: string;
-    label?: React.ReactNode;
+    label?: string;
```

Comment:
why do we need this change?

Reply:
https://github.com/amina-deriv/deriv-app/commit/0ff5b3743ddfb701015603ce94db740e991a24fd
cc: @likhith-deriv

Reply:
This is a common component. wouldnt this change effect whole codebase?

Reply:
No it will not affect. Because now we are enforcing it to be string and not a a React element or a string

Reply:
i tried setting the type to ReactNode, JSXElement and Element; but i think since the type involves `HTMLInputElement &`, it doesn't work.
<img width="711" alt="Screenshot 2023-11-27 at 3 57 22 PM" src="https://github.com/binary-com/deriv-app/assets/122449658/1c869599-33e1-46bb-9cd1-6640a31053e4">